tests/nativeapp/test_project_model.py

```diff
@@ -34,19 +32,24 @@
 )
 from snowflake.cli.api.project.util import TEST_RESOURCE_SUFFIX_VAR
 
+from tests.nativeapp.factories import PdfV10Factory
+
 CURRENT_ROLE = "current_role"
 
 
-@pytest.mark.parametrize("project_definition_files", ["minimal"], indirect=True)
+def use_minimal_project_definition():
+    pdf_res = PdfV10Factory(native_app__artifacts=["setup.sql", "README.md"])
+    return pdf_res.yml, pdf_res.path
+
+
 @mock.patch("snowflake.cli._app.snow_connector.connect_to_snowflake")
 @mock.patch.dict(os.environ, {"USER": "test_user"}, clear=True)
-def test_project_model_all_defaults(
-    mock_connect, project_definition_files: List[Path], mock_ctx
-):
+def test_project_model_all_defaults(mock_connect, mock_ctx, temp_dir):
     ctx = mock_ctx()
     mock_connect.return_value = ctx
-
-    project_defn = load_project(project_definition_files).project_definition
+    minimal_yml, pdf_path = use_minimal_project_definition()
```

Comment:
nit: so, it's just a fixture? You could just have the pdf_res above injected as a pytest fixture.
nit#2: generally not a fan of returning pairs that don't intuitively make sense, from a maintainability POV.

Reply:
use_minimal_project_definition can be changed into a fixture and used as such yes. But I preferred to keep the definition of the project close to the tests instead of in the fixtures file.    
- nit #2 solved

Reply:
You can define a fixture in the same file, FYI. However, in this case, it might make sense eventually to offer this as a reusable fixture since it's a pretty good starting point for anything that needs a PDF.

Reply:
To be clear, not a blocker for this PR though

Reply:
Made it a fixture in the same file.